handbook/engineering/search/index.md

```diff
@@ -53,6 +53,7 @@ Iterations start **every other Monday**.
 ## Members
 
 - We're hiring a [Product Manager](https://jobs.lever.co/sourcegraph/254299f5-f91b-43e2-aa1a-3732963dd296) for this role. [Christina Forney](../../../company/team/index.md#christina-forney-she-her) is involved in the meantime.
+- [Rob Rhyne](../../../company/team/index.md#rob-rhyne) ([Product Designer](../../product/roles/product_designer.md))
```

Comment:
Should @quinnkeast  be added here too (for code monitoring & search contexts)?

Reply:
@quinnkeast @rrhyne my initial idea was to have 1 designer per team. Should we add both of you here?

Reply:
@quinnkeast @rrhyne my initial idea was to have 1 designer per team. Should I add both of you to the search team?

Reply:
For clarity in the short term, it may be useful.